docs/source/CONTRIBUTE.md

```diff
@@ -1,9 +1,35 @@
-# Contributing to MALA
-
-MALA is open source software and as such always welcomes additions and
-improvements. However, we kindly ask any contributor to adhere to the following
-suggestions in order to keep the overall quality and maintainability of the
-code high.
+# Contributions
+
+MALA is open source software and based on the work of many people. 
+The MALA team always welcomes contributions, but kindly asks potential 
+contributors to adhere to the following suggestions in order to keep the 
+overall quality and maintainability of the code high.
+
+## MALA contributors
+
+Many people have contributed to MALA, and we are deeply thankful for it. 
```

Comment:
Many people have made valuable contributions to MALA, and we are immensely grateful for their support.